pyproject.toml

```diff
@@ -1,29 +1,32 @@
 [tool]
 [tool.poetry]
 name = "mojap-metadata"
-version = "1.2.2"
+version = "1.3.0"
 description = "A python package to manage metadata"
 license = "MIT"
 authors = ["MoJ Data Engineering <dataengineering@digital.justice.gov.uk>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4"
+python = ">=3.7.1,<3.10"
```

Comment:
Is this forced by pydbtools?

Reply:
Yup, got complained at by poetry, this was the solution